package-lock.json

```diff
@@ -38646,7 +38476,34 @@
         "node": ">=14"
       },
       "peerDependencies": {
-        "@opentelemetry/api": "^1.3.0"
+        "@opentelemetry/api": "^1.3.0",
+        "@opentelemetry/api-logs": "^0.49.1"
+      }
+    },
+    "plugins/node/opentelemetry-winston-transport": {
+      "name": "@opentelemetry/winston-transport",
+      "version": "0.0.1",
+      "extraneous": true,
+      "license": "Apache-2.0",
+      "dependencies": {
+        "@opentelemetry/api-logs": "^0.48.0",
+        "winston-transport": "4.*"
+      },
+      "devDependencies": {
+        "@types/mocha": "7.0.2",
+        "@types/node": "18.6.5",
+        "@types/sinon": "10.0.18",
+        "@types/triple-beam": "^1.3.2",
+        "mocha": "7.2.0",
+        "nyc": "15.1.0",
+        "rimraf": "5.0.5",
+        "sinon": "15.2.0",
+        "test-all-versions": "6.0.0",
+        "ts-mocha": "10.0.0",
+        "typescript": "4.4.4"
+      },
+      "engines": {
+        "node": ">=14"
```

Comment:
```suggestion
```

This is a vestigial entry from when this package was in the plugins/node/... dir.
Doing regular `npm install` doesn't clean it up.

Reply:
Wow, how do you catch this kind of things?, I was expecting npm install to always take care of issues like this one, manually removed now.

Reply:
> Wow, how do you catch this kind of things?

With all the npm update troubles we've been having (see https://github.com/open-telemetry/opentelemetry-js-contrib/issues/1917 and others), I've gotten into the habit of reading/skimming package-lock diffs. 

> I was expecting npm install to always take care of issues like this

Indeed, that would be nice.  I'm not enamoured with `npm` these days.